pkg/gofr/context.go

```diff
@@ -58,6 +61,33 @@ func (c *Context) Bind(i interface{}) error {
 	return c.Request.Bind(i)
 }
 
+// WriteMessageToSocket writes a message to the WebSocket connection associated with the context.
+// The data parameter can be of type string, []byte, or any struct that can be marshaled to JSON.
+// It retrieves the WebSocket connection from the context and sends the message as a TextMessage.
+func (c *Context) WriteMessageToSocket(data interface{}) error {
+	// Retrieve connection from context based on connectionID
+	conn := c.GetConnectionFromContext(c)
```

Comment:
This is confusion although syntactically they are correct.
```suggestion
	conn := c.Container.GetConnectionFromContext(c)
```

or 

```suggestion
    conn := c.GetConnectionFromContext(c.Context)
```

Reply:
done